<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaoComida</class>
 <widget class="QDialog" name="fm_gestaoComida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comida</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>711</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gestão Comida</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>281</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>260</width>
       <height>289</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditUpdtNome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Descrição</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEditUpdtDescricao"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonSalvarComida_2">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonExcluirComida_2">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>137</width>
       <height>68</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Filtro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditFiltrar"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFiltrar">
        <property name="styleSheet">
         <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
        </property>
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>74</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Filtar por:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonIdComida">
        <property name="text">
         <string>ID Comida</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonNome">
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Adicionar Comida</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>311</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nome</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditAddNome"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Descrição</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditAddDescricao"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButtonSalvarComida_1">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonExcluirComida_1">
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
